tensorflow/java/src/gen/cc/source_writer.cc

```diff
@@ -23,10 +23,17 @@ namespace tensorflow {
 namespace java {
 
 SourceWriter::SourceWriter() {
-  // push an empty generic namespace at start, for simplification
+  // Push an empty generic namespace at start, for simplification.
   generic_namespaces_.push(new GenericNamespace());
 }
 
+SourceWriter::~SourceWriter() {
+  // Remove empty generic namespace added at start.
```

Comment:
It seems like PushGenericNamespace and PopGenericNamespace always have to be called in pairs (pushing and popping a GenericNamespace to the stack). But the initial GenericNamespace that is pushed (the comment states "as a simplification") doesn't have a corresponding pop.

@karllessard, do you have any comments about this fix

Reply:
This seems to rely on user of SourceWriter always calling PopGenericNamespace whenever they called PushGenericNamespace. Since it is left to the class user, I think it is best to handle the case when is not called.

Reply:
@case540: you are right. In fact, like @annarev was pointing out, I think it would be safer to empty our stack in the destructor to delete any remaining namespace that might have been left 'opened' inadvertently, not only the default one.

Reply:
Done. Thanks!